test_automl/step2_cell_type_annotation_actinn_example.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any, Callable, Dict, Tuple
+
+import numpy as np
+import torch
+from step2_config import get_transforms, log_in_wandb, setStep2
+
+from dance import logger
+from dance.datasets.singlemodality import CellTypeAnnotationDataset
+from dance.modules.single_modality.cell_type_annotation.actinn import ACTINN
+from dance.transforms.misc import Compose
+from dance.utils import set_seed
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+@log_in_wandb(config=None)
+def train(config):
+
+    model = ACTINN(hidden_dims=config.hidden_dims, lambd=config.lambd, device=device)
+    transforms = get_transforms(config=config)
+    if transforms is None:
```

Comment:
When transform is None, it represents an exception, but I don't want to stop searching